packages/studio-ui/package.json

```diff
@@ -0,0 +1,73 @@
+{
+  "name": "@yext/studio-ui",
+  "version": "0.22.0",
+  "types": "./lib/src/index.d.ts",
+  "main": "./lib/src/index.js",
+  "type": "module",
+  "exports": {
+    ".": {
+      "types": "./lib/src/index.d.ts",
+      "import": "./lib/src/index.js",
+      "require": "./lib/src/index.js"
+    }
+  },
+  "files": [
+    "lib"
+  ],
+  "scripts": {
+    "dev": "npm run copy-svg && tsc --watch --preserveWatchOutput -p tsconfig.json",
+    "copy-svg": "npx copyfiles \"src/**/*.svg\" lib",
```

Comment:
for my _edification_, I saw that before we didn't include icons in lib-- why do we need them now/didn't we need them before?

Reply:
oh so typescript doesn't handle .svg imports at all, but we still need these svgs when running the dev build

https://github.com/microsoft/TypeScript/issues/30835